cmd/source/kubernetes/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
```

Comment:
That is because slack renderer has very special functionality to render kubernetes metadata like that, so rendering used to be handled on slack_renderer. However, in plugin structure, we render messages in advance, and I tried to unified them as a general message format. Again, I am open to suggestions here :)

Reply:
followup

Reply:
@mszostok is on it